freshmaker/lightblue.py

```diff
@@ -1621,6 +1621,10 @@ def find_images_to_rebuild(
             rpm_nvrs, content_sets, filter_fnc, published,
             release_categories, leaf_container_images=leaf_container_images)
 
+        # Create a new list of images which contain non-hotfix images
+        is_hotfix_img = any(label["name"] == "com.redhat.hotfix" for label in images[0]["parsed_data"]["labels"])
```

Comment:
iirc, the images here are images of all NVRs, each one represent a NVR with only one arch, you're checking the first image instead of all images. I'd prefer you submit another PR for CWFHEALTH-1132.

Reply:
I actually had just noticed this as I pushed it up. Essentially, I have to remove the [0] correct?

And sure I can submit a separate PR for this

Reply:
>  remove the [0]

You need to check labels in each images, it should be something like:
```
images = [
    img
    for img in images
    if any(label["name"] == "com.redhat.hotfix" for label in img["parsed_data"]["labels"])
]
```
but I didn't test it, you need to test and verify this with the images data.

Reply:
no problem, I'll write the code to check each image. I assigned CWFHEALTH-1132 to myself and will work on that after this  PR gets merged